test/packages/bad_deployment_mode_without_identities/manifest.yml

```diff
@@ -0,0 +1,49 @@
+format_version: 3.1.4
+name: bad_deployment_mode
```

Comment:
For the package to be used in tests it must be added to the list of packages here: https://github.com/elastic/package-spec/blob/5f2c9ee5976c4833d3fe9791e5a616b369151fae/code/go/pkg/validator/validator_test.go#L28